pkg/porter/lifecycle_test.go

```diff
@@ -160,6 +162,25 @@ func TestBundleLifecycleOpts_ToActionArgs(t *testing.T) {
 	})
 }
 
+func TestManifestIgnoredWithTag(t *testing.T) {
+	p := NewTestPorter(t)
+	t.Run("ignore manifest in cwd if tag present", func(t *testing.T) {
+		opts := BundleLifecycleOpts{}
+		opts.Tag = "deislabs/kubekahn:latest"
+
+		wd, _ := os.Getwd()
+		// Below line makes cnab.go#defaultBundleFiles#manifestExists true
+		p.TestConfig.TestContext.AddTestFileContents([]byte(""), path.Join(wd, config.Name))
+		// When execution reach to `readFromFile`, manifest file path will be lost.
+		// So, had to empty default manifest content also
+		// Below line makes manifest.go#readFromFile#exists true
+		p.TestConfig.TestContext.AddTestFileContents([]byte(""), config.Name)
```

Comment:
I couldn't make this test fail with `main` branch with just one empty manifest file. 

`defaultBundleFiles#manifestExists` will be `true` only if I use `path.Join(wd....` and we need it to execute bundle default logic.
But `readFromFile` func gets only filename (not complete path) and reads  manifest file from root directory. So, I had to use root manifest file also

Reply:
Ah I see.  The in-line comments help!  Thanks for updating.